tests/tests_ecg_findpeaks.py

```diff
@@ -23,6 +24,23 @@ def _read_csv_column(csv_name, column):
     csv_data = pd.read_csv(csv_path, header=None)
     return csv_data[column].to_numpy()
 
+def test_ecg_findpeaks_all_methods_handle_empty_input():
+    METHODS = ["neurokit", "pantompkins", "nabian", "gamboa", 
+               "slopesumfunction", "wqrs", "hamilton", "christov",
+               "engzee", "manikandan", "elgendi", "kalidas", 
+               "martinez", "rodrigues"]
```

Comment:
If you decide to leave out the `"vgraph"` method for this PR, I would suggest specifying in a comment why you are not including it, maybe with a link to the comment in the PR

Reply:
comment added and opened #1009 to track the ts2vg issue.